### DISCUSSION TITLE
Is there a way to get the timestamps of recently sent messages except from the output of "send" command?

### DISCUSSION THREAD

**Discussion by feb-ri** [AsamK/signal-cli#1157] [Q&A] [answered]
Timestamp is required to delete a message. In case we forgot to save the timestamp from the output of the "send" command, is there another way to get it? I read manual page and the feature matrix, there didn't seem to be such a feature.

Thanks.

**Comment by AsamK** [comment, marked as answer]
No, you only get the timestamp from the output of the send command.

**Reply by feb-ri** [reply]
Ah, I see. Thank you.
